src/components/ItemList.jsx

```diff
@@ -1,16 +1,19 @@
-import icHeart from "../assets/ic_heart.svg";
+import { useNavigate } from "react-router-dom";
+import icHeart from "../assets/heart/ic_heart_small_inactive.svg";
 import styles from "./ItemList.module.css";
 
 export default function ItemList ( { rows, columns, items } ) {
+    const navigate = useNavigate();
+
     return (<section className={styles.list} style={{gridTemplate: `repeat(${rows}, 1fr) / repeat(${columns}, 1fr)`}}>
         {items.slice(0, rows*columns).map((e) => (
-            <Item key={e.id} {...e} />
+            <Item key={e.id} onClick={() => navigate("/items/"+e.id)} {...e} />
         ))}
     </section>);
 }
 
-function Item ( { name, price, favoriteCount, images } ) {
-    return (<article className={styles.item}>
+function Item ( { name, price, favoriteCount, images, onClick } ) {
+    return (<article className={styles.item} onClick={onClick}>
```

Comment:
❗️ 수정요청
이런 클릭 가능한 요소의 경우 적절한 태그를 사용해주세요. 더 적절한 태그가 있는데 아티클 태그에 onClick 을 주게되면 클릭 요소에게 기대하는 동작을 제대로 수행하지 못합니다!

```suggestion
    return (
      <Link className={styles.item} to={href}> // 지금과 같은 경우 컴포넌트 클릭 시 페이지 이동이니 Link 태그를 써주세요.
```